documentation/ci-fix.md

```diff
@@ -44,3 +44,22 @@ oav validate-spec <swagger-spec-path>
 Please see [readme](https://github.com/Azure/oav/blob/master/README.md) for how to install or run tool in details.
 Or you can run it in [OpenAPI Hub](https://portal.azure-devex-tools.com/tools/static-validation/static/errors/default)
 Refer to [Semantic and Model Violations Reference](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/Semantic-and-Model-Violations-Reference.md) for detailed description of validations and how-to-fix guidance.
+
+## Avocado
+
+Run avocado locally:
+
+```
+npm install -g @azure/avocado
+
```

Comment:
Should we add step here,?e.g. cd <your RP directory>

Reply:
No. `npm install -g` mean install in root node_module. You can use avocado in any directory.

Reply:
I mean cd to the RP folder before run avocado

Reply:
Currently, Avocado can't support running in RP folder.  Avocado running process based on root folder.  I can design a feature to support it.